library/item/item_extensions/src/main/java/org/quiltmc/qsl/item/extensions/mixin/client/AbstractClientPlayerEntityMixin.java

```diff
@@ -0,0 +1,27 @@
+package org.quiltmc.qsl.item.extensions.mixin.client;
```

Comment:
```suggestion
/*
 * Copyright 2023 The Quilt Project
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

package org.quiltmc.qsl.item.extensions.mixin.client;
```

Reply:
relocated file to org.quiltmc.qsl.item.extensions.mixin.bow.client and added license.